src/org/armedbear/lisp/disassemble.lisp

```diff
@@ -32,6 +32,20 @@
 (in-package :system)
 (require :clos)
 
+(eval-when  (:load-toplevel)
+  (defvar *available-disassemblers*
+	(let ((files 
+		(directory (merge-pathnames
```

Comment:
One cannot assume the presences of a the ABCL-INTROSPECT in the core implementation. Additionally, the presence of the function in a file does not necessarily mean the assembler is available, as it might have unsatisfied Maven load dependencies.

Reply:
If there aren't any disassemblers then *available-disassemblers* will be nil. It's a directory probe which simply fails. The require loads the appropriate system, which resolves the maven dependencies.